silence the MSVC env setup script

dwenegar opened the pull request:
The batch files used to initialize the environment when using the MSVC compiler print a banner that makes the use of `luarocks path` difficult in a CI context.

This PR make it so that the output of the environment initialization scripts is redirected to NUL.

hishamhm commented:
@dwenegar Thanks for the patch! Looks like your patch changed all of the Windows line-endings. (The Github diff shows all lines as modified) — could you double-check that? Thanks!!

dwenegar commented:
> 
> 
> @dwenegar Thanks for the patch! Looks like your patch changed all of the Windows line-endings. (The Github diff shows all lines as modified) — could you double-check that? Thanks!!

It did indeed change all the line endings.
It actually *fixed* them, since `.editorconfig` has `end_of_line = lf` for all files.

I updated the `.editrconfig` and fixed the line-endings.

hishamhm commented:
@dwenegar Thank you for going above and beyond! :) 

Merging!!
